### PR TITLE
added releasenotes for Milestone-1

### PR DESCRIPTION
**Pull request by vlvanchin**
Signed-off-by: vlvanchin <vanchinathan.lokanathan@bosch-si.com>

### CODE REVIEW

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Release Notes
+---
+{% include base.html %}
+## Release Notes - 0.10.0-M1
+
+We are proud to announce the release of 0.10.0-M1.
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Do not mention release! Instead write sth like: "We are proud to announce the milestone 0.10.0-M1 of Eclipse Vorto.

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+---
+{% include base.html %}
+## Release Notes - 0.10.0-M1
+
+We are proud to announce the release of 0.10.0-M1.  
+Here are some of the major highlights:
+
+- **Web Editor** that allows to create information, function blocks and entity type models through the web browser
+- New **Code Generators** have been added to the list of existing generators.
+- Bug fixes and improvements
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Many bug fixes and improvements

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
+### Web Editor
+With the **Web Editor**, you can create individual projects where you can describe the characteristics and functionality of the Information Models and publish then to Vorto Repository for others who could use to generate code using Code Generators. This is solely done using a web browser.
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: With the web editor, device manufacturers can describe devices online and publish these as information models to the Vorto Repository.

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+  ![WebEditor]({{base}}/img/documentation/webeditor_details.png){:width="700px"}
+  
+### Code Generators
+
+We have three new Code Generators out of the box, those are
+
+* Bosch IoT Suite
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Alexa Skill set for Bosch IoT Suite
Device Simulator for Bosch IoT Suite

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  
+#### Eclipse Kura
+
+This **Eclipse Kura** generator allows you to choose and generate client code for **Drivers (eg. Bluetooth)** and **Cloud Platform Connector for Bosch IoT Suite** types.
+
+  ![Generators]({{base}}/img/documentation/eclipse-kura_generator_configuration.png){:width="700px"}
+
+#### Web-based Device Dashboard
+
+This **Web-based Device Dashboard** generator allows you to choose the generate client code for **Historical Data** along with **Swagger Documentation" and , **Alexa Skillset** and **Cloud Platform Connector for Bosch IoT Suite** types
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: The Web - based Device Dashboard generator creates a Spring Boot Web application that is able to visualise the device data with various UI components as well as consume the data from a IoT Cloud backend.

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+#### Bosch IOT Suite
+
+This **Bosch IOT Suite** generator allows you to choose and  generate client code for **Device Simulator**, **Alexa Skillset** and **Device Validation** types.
+
+  ![Generators]({{base}}/img/documentation/bosch-iot_generator_configuration.png){:width="700px"}
+  
+#### Eclipse Kura
+
+This **Eclipse Kura** generator allows you to choose and generate client code for **Drivers (eg. Bluetooth)** and **Cloud Platform Connector for Bosch IoT Suite** types.
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Eclipse Kura generator supports the generation of a Kura gateway OSGI bundle that reads device data via bluetooth LE technology and sends the data to either Kura built-in Cloud Services or the Bosch IoT Suite.

File: website/documentation/appendix/releasenotes/0.10.0-M1.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+We have three new Code Generators out of the box, those are
+
+* Bosch IoT Suite
+* Eclipse Kura
+* Web-based Device Dashboard
+
+#### Bosch IOT Suite
+
+This **Bosch IOT Suite** generator allows you to choose and  generate client code for **Device Simulator**, **Alexa Skillset** and **Device Validation** types.
```
Comment (aedelmann) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: The Alexa Skill set Generator for Bosch IoT Suite generates an Alexa Skill Interaction model and Lambda function that fetches the device data from Bosch IoT Suite.
The Device Simulator Generator for Bosch IoT Suite generates a simulator that creates test data for a given information model and publishes the data to the Bosch IoT Suite.

**Comment by aedelmann** [comment]
@vlvanchin  Can you create a screenshot of the Web editor containing an actual device. Please take the screenshot from the tutorial in GitHub as it already describes a specific device as an example: https://github.com/eclipse/vorto/blob/development/tutorials/tutorial-create_and_publish_with_web_editor.md

**Comment by aedelmann** [comment]
Release Notes should also contain the Repository Java Client with the description:
"With the Repository Java Client you can find information models and process its content as well as generate code for various IoT platforms.

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 1 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the sentence

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 2 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the sentence

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 3 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the sentence

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 4 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the sentence

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 5 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the paragraph

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 6 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the paragraph

File: website/documentation/appendix/releasenotes/0.10.0-M1.md (hunk 7 above)
Comment (vlvanchin) [inline website/documentation/appendix/releasenotes/0.10.0-M1.md]: Amended the paragraph

**Pull request merged**